pkg/kube/inject/webhook.go

```diff
@@ -590,6 +591,9 @@ func postProcessPod(pod *corev1.Pod, injectedPod corev1.Pod, req InjectionParame
 }
 
 func applyMetadata(pod *corev1.Pod, injectedPodData corev1.Pod, req InjectionParameters) {
+	if nw, ok := req.proxyEnvs["ISTIO_META_NETWORK"]; ok {
```

Comment:
Was this exact code accidentally removed previously?

Reply:
Close to it: 

https://github.com/howardjohn/istio/blob/01a3e0141ce3c8a89fb3045fb09b716c345b3d9b/pkg/kube/inject/webhook.go#L545

Reply:
Ok cool ... before you submit, mind adding the details of what happened to the description?